### DISCUSSION TITLE
Q: IP属地能只具体到省吗？ || Q: Can IP territories be specific to provinces only?

### DISCUSSION THREAD

**Discussion by haloged** [walinejs/waline#2010] [Q&A] [answered]
### Describe the bug | 问题描述

IP属地能只具体到省吗？
![QQ截图20230622220438](https://github.com/walinejs/waline/assets/101408211/e9682259-5964-4112-bbe4-2b97f24d9904)

### Website URL | 问题网站

https://haloged.eu.org/posts/3fe50ae2

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

Can the IP territory be specific only to the province?
![QQ screenshot 20230622220438](https://github.com/walinejs/waline/assets/101408211/e9682259-5964-4112-bbe4-2b97f24d9904)

### Website URL | Problem Website

https://haloged.eu.org/posts/3fe50ae2

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
这个要看具体的 IP和地理位置数据，Waline 没有这个限制。

---
This depends on the specific IP and geographic location data, Waline does not have this limitation.
